packages/react-components/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -251,7 +256,8 @@ export const ActionMenu: React.FC<ActionMenuProps> = ({
   return (
     <>
       <div
-        data-testid="action-menu-trigger-button"
```

Comment:
If it was passed before, shouldn't it stay and be overwritten by custom data-test id if passed?

Reply:
Its set as default value in props